src/ui/components/pages/otherAccounts.tsx

```diff
@@ -60,6 +61,22 @@ export function OtherAccountsPage() {
     <div className={styles.root} data-testid="otherAccountsPage">
       <header className={styles.header}>
         <h2 className={styles.title}>{t('otherAccounts.title')}</h2>
+
+        <button
+          className={clsx(
+            styles.addAccountButton,
+            styles.addAccountButtonSmall
+          )}
+          data-testid="addAccountButton"
+          type="button"
+          onClick={() => {
```

Comment:
Better extract this function than duplicating. I doubt it would ever be different on one of these buttons, so it makes sense just to reuse this code. Doesn't matter if it's small, it's just meant to be the same.